cmake/SetupPch.cmake

```diff
@@ -50,6 +50,9 @@ target_link_libraries(
   hdf5::hdf5
   SpectreFlags
   )
+if(TARGET OpenMP::OpenMP_CXX)
+  target_link_libraries(SpectrePchFlags INTERFACE OpenMP::OpenMP_CXX)
```

Comment:
Have you measured the overhead of this?  I would not expect running OpenMP sections with a single thread to be as fast as compiling them without OpenMP, but I don't know if the change is significant for us.

Reply:
I added a benchmark with some results. I didn't find any measurable overhead when running on a single thread with OpenMP linked.

Reply:
Sounds good, then.